packages/video_player/android/src/main/java/io/flutter/plugins/videoplayer/VideoPlayerPlugin.java

```diff
@@ -252,6 +263,18 @@ private VideoPlayerPlugin(Registrar registrar) {
 
   private final Registrar registrar;
 
+  void onDestroy() {
+    // The whole FlutterView is being destroyed. Here we release resources
+    // acquired for all instances of VideoPlayer.
+    // TODO(https://github.com/flutter/flutter/issues/20989) Replace with
```

Comment:
syntax for TODOs is `TODO(username): description, url`

Reply:
I rephrased this in another way, so that I'm not on the hook, but the bug owner :).